docs/en/dev_setup/dev_env_mac.md

```diff
@@ -21,17 +21,6 @@ The "base" macOS setup installs the tools needed for building firmware, and incl
 
 ### Environment Setup
 
-:::details Apple Silicon Macbook users!
```

Comment:
Does this means that the standard setup works for all types of silicon?

Reply:
yes it works for all!